doc/source/cluster/vms/user-guides/launching-clusters/on-premises.md

```diff
@@ -101,7 +101,7 @@ Test that it works by running the following commands from your local machine:
 # Download the example-full.yaml
 wget https://raw.githubusercontent.com/ray-project/ray/master/python/ray/autoscaler/local/example-full.yaml
 
-# Update the example-full.yaml to update head_ip, worker_ips, and ssh_user.
+# Update the example-full.yaml to update head_ip, worker_node_ips, and ssh_user.
```

Comment:
```suggestion
# Update the example-full.yaml to update head_node_ip, worker_node_ips, and ssh_user.
```

Same question as above

Reply:
Yeah, it's not as confusing as the worker situation, but it would be better to be consistent.  Perhaps in a followup PR so as not to make this one too large?